src/wavesurfer.js

```diff
@@ -894,6 +894,16 @@ export default class WaveSurfer extends util.Observer {
         return this.isMuted;
     }
 
+    /**
+     * Get the current ready status.
+     *
+     * @example const isReady = wavesurfer.isReady();
+     * @return {boolean}
+     */
+    isReady() {
+        return this.isMuted;
```

Comment:
@Guisch what does `isMuted` have to do with this? :)

Reply:
Ahah, thanks Copy/Paste